modules/apps/headless/headless-delivery/headless-delivery-test/src/testFunctional/macros/DocumentFolderAPI.macro

```diff
@@ -66,6 +101,60 @@ definition {
 		}
 	}
 
+	macro assertProperDocumentFolderTotalCount {
+		Variables.assertDefined(parameterList = ${expectedDocumentTotalCount});
```

Comment:
I only see one parameter here when in the macro there are 4 parameters needed for it to work.